### PR TITLE
chore: fix-side-effects

### PR DESCRIPTION
**Pull request by ryanbas21**
add side effects array to the package.json for sdk&protect.

# JIRA Ticket

Please link jira ticket here

# Description

# Type of Change

Please Delete options that are not relevant

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Definition of Done

Check all that apply

- [ ] Acceptance criteria is met.
- [ ] All tasks listed in the user story have been completed.
- [ ] Coded to standards.
- [ ] Code peer-reviewed.
- [ ] Ensure backward compatibility (special attention).
- [ ] API reference docs is updated.
- [ ] Unit tests are written.
- [ ] Integration tests are written.
- [ ] e2e tests are written.
- [ ] CI build passing on the feature branch.
- [ ] Functional spec is written/updated
- [ ] contains example code snippets.
- [ ] Change log updated.
- [ ] Documentation story is created and tracked.
- [ ] UI is completed or ticket is created.
- [ ] Demo to PO and team.
- [ ] Tech debts and remaining tasks are tracked in separated ticket(s).

## Documentation

- [ ] Acceptance criteria met
- [ ] Spell-check run
- [ ] Peer reviewed
- [ ] Proofread

### CODE REVIEW

File: packages/javascript-sdk/vite.config.ts [hunk 1]
```diff
@@ -31,11 +31,13 @@ export default defineConfig({
     dts({
       declarationOnly: false,
       entryRoot: 'src',
-      rollupTypes: true,
+      rollupTypes: false,
```
Comment (ryanbas21) [inline packages/javascript-sdk/vite.config.ts:34]: this was breaking the src/* imports.

File: packages/ping-protect/package.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "url": "git+https://github.com:ForgeRock/forgerock-javascript-sdk.git",
     "directory": "packages/ping-protect"
   },
+  "sideEffects": ["./dist/lib/ping-signals-sdk.js"],
```
Comment (ryanbas21) [inline packages/ping-protect/package.json:14]: since this file performs some effects on the window we should list it.

File: package.json [hunk 3]
```diff
@@ -119,6 +119,7 @@
   },
   "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e",
   "dependencies": {
+    "browserstack-node-sdk": "1.34.17",
```
Comment (ryanbas21) [inline package.json:122]: i left this by mistake but its not a big deal i dont wanna back it out for the sake of this pr, i will fix this going forward

**Review by george-bafaloukas-forgerock — APPROVED** [review]
LGTM

**Review by jeyanthanperiyasamy — APPROVED** [review]

**Review by spetrov — APPROVED** [review]

File: packages/javascript-sdk/package.json [hunk 4]
```diff
@@ -27,14 +28,14 @@
         "default": "./dist/index.cjs"
       }
     },
-    "./src/*": {
```
Comment (ryanbas21) [inline packages/javascript-sdk/package.json:30]: have to add this back, my mistake

**Pull request merged**